src/pages/HomePage/HomePage.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { ResponsiveContainer } from '@/components/ResponsivePage/ResponsiveContainer';
+import { Sidebar, SidebarProps } from '@/components/Sidebar/Sidebar';
+import { NavBar } from '@/components/NavBar/NavBar';
+
+// TODO: Remove once fetch calls in place
```

Comment:
I thought we were saying we are now using the actual fetch calls from the API no? Or am I confusing that with something else?

Reply:
Yes we are. That will be in a separate PR as this one is mainly for the responsive container component and other styling changes.